### PR TITLE
refactor(setup-dev-env.sh): make the target playbook configurable

### PR DESCRIPTION
**Pull request by kenji-miyake**
## Description

Considering that users fork this repository and customize the configurations depending on their system, it is useful to change the meaning of the argument.

With this change, users can do like the following:

```bash
# Add some roles for their system
vim ~/autoware.customized/ansible/roles
vim ~/autoware.customized/ansible/playbooks/customized_playbook.yaml

# Replace the base name
sed -e "s/autoware.dev_env/autoware_customized.dev_env/g" setup-dev-env.sh

# Call the playbook
./setup-dev-env.sh customized_playbook
```

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by kenji-miyake** [comment]
@xmfcx @esteve @yukkysaito Could you review this, please? :pray:

File: setup-dev-env.sh [hunk 1]
```diff
@@ -105,8 +100,8 @@ fi
 ansible-galaxy collection install -f -r "$SCRIPT_DIR/ansible-galaxy-requirements.yaml"
 
 # Run ansible
-echo Run ansible-playbook "autoware.dev_env.$installation_type" "${ansible_args[@]}"
-if ansible-playbook "autoware.dev_env.$installation_type" "${ansible_args[@]}"; then
+echo -e "\e[36m Run ansible-playbook" "$target_playbook" "${ansible_args[@]}" "\e[m"
```
Comment (xmfcx) [inline setup-dev-env.sh:103]: What are `\e[36m` and `\e[m` here?

Comment (kenji-miyake) [reply setup-dev-env.sh:103]: Just to colorize the console output message!

Comment (kenji-miyake) [reply setup-dev-env.sh:103]: image

**Review by xmfcx — APPROVED** [review]

**Pull request merged**